### PR TITLE
RP Cloud does not support RF=1

### PR DESCRIPTION
**Pull request by Feediver1**
Fixes https://github.com/redpanda-data/documentation-private/issues/2156

[Preview](https://deploy-preview-376--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/create-topic/)

### CODE REVIEW

File: modules/deploy/pages/deployment-option/cloud/create-topic.adoc [hunk 1]
```diff
@@ -20,7 +20,7 @@ After you create a cluster, you can create a topic for that cluster.
 
 | *Replication factor*
 | The number of partition replicas for the topic. Three replicas are recommended. The value for this property must meet or exceed the cluster's xref:reference:cluster-properties.adoc#minimum_topic_replication[`minimum_topic_replication`] setting. 
-NOTE: Running Redpanda Cloud topics with a replication factor of one is _not_ recommended. If you do so, and data is lost, Redpanda is unable to assist with recovery. 
+**NOTE:** Running Redpanda Cloud topics with a replication factor of one is _not_ recommended. If you do so, and data is lost, Redpanda is unable to assist with recovery.
```
Comment (Deflaimun) [inline modules/deploy/pages/deployment-option/cloud/create-topic.adoc]:
```suggestion

NOTE: Running Redpanda Cloud topics with a replication factor of one is _not_ recommended. If you do so, and data is lost, Redpanda is unable to assist with recovery.

```

**Comment by micheleRP** [comment]
This covers the original issue, but should we document https://github.com/redpanda-data/documentation-private/issues/2314 with this?

**Comment by micheleRP** [comment]
@Feediver1 looking at this page now, I don't think we need to document the steps for creating a topic as steps 1-6. It's intuitive in the UI and this makes it look harder than it is. Maybe we just say that within your cluster, you can create topics, and Dedicated and BYOC clusters allow you to overwrite default settings. If you keep the steps, please change the first one to: "From the Clusters page, select the cluster where you want to create a topic." With the new nav, there is no home page (users will land on the Clusters page).

**Comment by micheleRP** [comment]
At least for replication factor, maybe we should say that Redpanda recommends not changing the default value of 3, not decreasing it or increasing it. It sounds like raising the RF could also cause issues. See https://redpandadata.slack.com/archives/C02JDTMG8GN/p1711545806387389

File: modules/deploy/pages/deployment-option/cloud/create-topic.adoc [hunk 2]
```diff
@@ -20,7 +21,8 @@ After you create a cluster, you can create a topic for that cluster.
 
 | *Replication factor*
 | The number of partition replicas for the topic. Three replicas are recommended. The value for this property must meet or exceed the cluster's xref:reference:cluster-properties.adoc#minimum_topic_replication[`minimum_topic_replication`] setting. 
-NOTE: Running Redpanda Cloud topics with a replication factor of one is _not_ recommended. If you do so, and data is lost, Redpanda is unable to assist with recovery. 
+
+IMPORTANT: While it is possible to modify the replication factor, Redpanda recommends that you do not change the replication factor value without discussing potential outcomes with your Redpanda support or customer success contact. In particular, updating the value for running Redpanda Cloud topics with a replication factor of one is _not_ recommended. If you do so, and data is lost, Redpanda is unable to assist with recovery.
```
Comment (Deflaimun) [inline modules/deploy/pages/deployment-option/cloud/create-topic.adoc]:
```suggestion
IMPORTANT: While it is possible to modify the replication factor, Redpanda recommends that you do not change the replication factor value without discussing potential outcomes with your Redpanda support or customer success contact. In particular, updating the value for running Redpanda Cloud topics with a replication factor of one is _not_ recommended. If you do so, and data is lost, Redpanda is unable to assist with recovery. 
```

**Review by wzzzrd86 — APPROVED** [review]
Looks good and clearly visible.

**Comment by Deflaimun** [comment]
should we add something stating that rf factor must be odd?

**Review by kbatuigas — APPROVED** [review]
I've gone ahead and approved this, but I think Paulo's suggestion about RF being on odd number might be worth looking into

**Review by micheleRP — APPROVED** [review]
lgtm

File: modules/deploy/pages/deployment-option/cloud/create-topic.adoc [hunk 3]
```diff
@@ -3,15 +3,9 @@
 :page-aliases: cloud:create-topic.adoc
 :page-cloud: true
 
-After you create a cluster, you can create a topic for that cluster.
+After creating a cluster, you can create a topic for that cluster on the *Clusters* page, where you select the cluster in which you want to create the topic. For Dedicated and BYOC clusters, you can optionally overwrite the default settings. Defaults are populated from information stored in the broker.
```
Comment (micheleRP) [inline modules/deploy/pages/deployment-option/cloud/create-topic.adoc]:
```suggestion
After creating a cluster, you can create a topic for that cluster. Topic properties are populated from information stored in the broker. For Dedicated and BYOC clusters, you can optionally overwrite the default settings. 
```

File: modules/deploy/pages/deployment-option/cloud/create-topic.adoc [hunk 4]
```diff
@@ -20,7 +14,8 @@ After you create a cluster, you can create a topic for that cluster.
 
 | *Replication factor*
 | The number of partition replicas for the topic. Three replicas are recommended. The value for this property must meet or exceed the cluster's xref:reference:cluster-properties.adoc#minimum_topic_replication[`minimum_topic_replication`] setting. 
-NOTE: Running Redpanda Cloud topics with a replication factor of one is _not_ recommended. If you do so, and data is lost, Redpanda is unable to assist with recovery. 
+
+IMPORTANT: While it is possible to modify the replication factor, Redpanda recommends that you do not change the replication factor value without discussing potential outcomes with your Redpanda support or customer success contact. In particular, updating the value for running Redpanda Cloud topics with a replication factor of one is _not_ recommended. If you do so, and data is lost, Redpanda is unable to assist with recovery.
```
Comment (micheleRP) [inline modules/deploy/pages/deployment-option/cloud/create-topic.adoc]:
```suggestion
IMPORTANT: Redpanda recommends that you do not change the replication factor value without discussing potential outcomes with your Redpanda support or customer success contact. In particular, updating the value for running Redpanda Cloud topics to a replication factor of one is _not_ recommended. If you do so, and data is lost, Redpanda is unable to assist with recovery. 
```

**Pull request merged**